### PR TITLE
Add option to pass pathPrefix to common fetch function

### PR DESCRIPTION
**Pull request by karelhala**
### Additional API enpodints support

Since we might want to query multiple APIs let's add option to `commonFetch` to specify the API path partial.

### CODE REVIEW

File: frontend/src/sdk/commonFetch.ts [hunk 1]
```diff
@@ -63,13 +63,17 @@ export const commonFetch =
...
+
     try {
       let safeURL = url;
       if (/^\/\//.test(url)) {
         // https://github.com/openshift/dynamic-plugin-sdk/pull/55
         safeURL = url.slice(1);
       }
-      return validateStatus(await fetch(new Request(`${k8sBasePath}${safeURL}`, { credentials: 'include' }), allOptions));
+      return validateStatus(
+        await fetch(new Request(`${`${prefix.indexOf('/') === 0 ? '' : '/'}${prefix}`}${safeURL}`, { credentials: 'include' }), allOptions),
```
Comment (florkbr) [inline frontend/src/sdk/commonFetch.ts:75]: @karelhala small NIT - but can we perform the leading `/` check on line 66 where we initialize the prefix for readability?

Comment (karelhala) [reply frontend/src/sdk/commonFetch.ts:75]: Sure thing!

**Comment by vidyanambiar** [comment]
Looks good!

**Pull request merged**